.github/workflows/wildfly-build.yml

```diff
@@ -0,0 +1,78 @@
+# This workflow is for a nightly run against WildFly upstream
+
+name: WildFly Test Build
+
+on:
+  push:
+    branches:
+      - 'main'
```

Comment:
@rsearls  I remembered we probably don't need this push settings because schedule job only works on default branch. Can you please check this again ?

Reply:
Done